plugins/modules/info.py

```diff
@@ -2980,186 +3358,177 @@ def get_nfs_global_settings(self):
             LOG.error(error_msg)
             self.module.fail_json(msg=error_msg)
 
+    def get_support_assist_settings(self):
+        """Get support assist settings based on the version."""
+        if self.major > 9 or (self.major == 9 and self.minor > 4):
+            return SupportAssist(self.support_assist_api, self.module).get_support_assist_settings()
+        else:
```

Comment:
If the array does not support, then we should return an empty dict instead of returning an error.

Reply:
Hi @anupamaloke,
Team will fix these 2 comments in next release sprint.